tests/common/util/TestHttpServer.hpp

```diff
@@ -43,7 +43,7 @@ class TestHttpServer {
      * @param host host to bind to
      * @param port port to bind to
```

Comment:
This will later trigger doxygen to error out.. best removed now since the param is already gone 👍